dask_match/expr.py

```diff
@@ -696,6 +696,29 @@ class NE(Binop):
     _operator_repr = "!="
 
 
+class Partitions(Expr):
+    """Select one or more partitions"""
+
+    _parameters = ["frame", "partitions"]
+
+    @property
+    def _meta(self):
+        return self.frame._meta
+
+    def _divisions(self):
+        divisions = []
+        for part in self.partitions:
+            divisions.append(self.frame.divisions[part])
+        divisions.append(self.frame.divisions[part + 1])
+        return tuple(divisions)
+
+    def _layer(self):
+        return {
+            (self._name, i): (self.frame._name, part)
+            for i, part in enumerate(self.partitions)
+        }
```

Comment:
Can I ask for a small test for this?

Reply:
Sure - Do you have specific functionality in mind that isn’t covered by test_partitions?

Reply:
Oh I'm sorry, I did a cursory look and didn't see anything.  Whoops.